packages/apollo-client/src/core/QueryManager.ts

```diff
@@ -1403,6 +1403,10 @@ export class QueryManager<TStore> {
   }
 
   public stopPollingQuery(queryId: string) {
+    const info = this.pollingInfoByQueryId.get(queryId)
+    if (info){
+      clearTimeout(info.timeout)
```

Comment:
```suggestion
    const info = this.pollingInfoByQueryId.get(queryId);
    if (info) {
      clearTimeout(info.timeout);
```
Just some nits that I'll merge myself.